### DISCUSSION TITLE
/opt/homebrew/bin/ getting removed after sometime

### DISCUSSION THREAD

**Discussion by tomdiers** [Homebrew/discussions#4928] [Getting started]
### Output of `brew config`

```shell
zsh: command not found: brew
```

### Output of `brew doctor`

```shell
zsh: command not found: brew
```

### Description of issue

I have the issue that the executable of Homebrew disappears after some time. The entire 'bin' folder in the Homebrew directory is removed, and I don't know why. I am using an M1 Max which was set up at the beginning of the M1 Macs era. Everything else is still in the Homebrew folder, and if I reinstall Homebrew, everything works for some time. Then the problem reappears.

**Comment by SMillerDev** [comment]
Is your device managed by an IT department? Do you have an antivirus running that could have misidentified brew?

**Reply by tomdiers** [reply]
So we have Sophos but there is no log for detecting that file. If i copy the file to an different folder, it is not deleted.
